### PR TITLE
Clean dates bigquery

### PR DESCRIPTION
**Pull request by GJMcGowan**

### CODE REVIEW

File: api/dbadapters/bigquery.ts [hunk 1]
```diff
@@ -134,12 +134,37 @@ export class BigQueryDbAdapter implements DbAdapter {
...
     }
   }
+
+  private cleanRows(rows: any[]) {
+    if (rows.length === 0) {
+      return rows;
+    }
+
+    const sampleData = rows[0];
+    const BIGQUERY_DATE_CLASS_NAME = "BigQueryDate";
```
Comment (BenBirt) [inline api/dbadapters/bigquery.ts]: extract this to the top of the file plz?

File: api/dbadapters/bigquery.ts [hunk 2]
```diff
@@ -134,12 +134,37 @@ export class BigQueryDbAdapter implements DbAdapter {
...
+    const fieldsWithBigQueryDates = Object.keys(sampleData).filter(
+      key =>
+        sampleData[key] &&
+        sampleData[key].constructor &&
+        sampleData[key].constructor.name === BIGQUERY_DATE_CLASS_NAME
+    );
+    if (fieldsWithBigQueryDates.length === 0) {
```
Comment (BenBirt) [inline api/dbadapters/bigquery.ts]: suggest this which might be a little simpler:

```
fieldsWithBigQueryDates.forEach(dateField => {
  rows.forEach(row => row[dateField] = row[dateField].value);
});
```

Comment (GJMcGowan) [reply api/dbadapters/bigquery.ts]: Fair enough, changed. I was vaguely thinking of not mutating the original variable but I don't think it matters too much.

**Comment by GJMcGowan** [comment]
@BenBirt I've updated the version in common.package.json - is there anywhere else I need to update the version before I bring it into `dataform-co`?

**Pull request merged**